index.html

```diff
@@ -2,19 +2,77 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta name="description" content="Your description goes here">
+    <meta name="Relaxr" content="Relaxr Page 2">
     <meta name="keywords" content="one, two, three">
 
-    <title>Blank Template</title>
+    <title>Relaxr Blog</title>
 
     <!-- external CSS link -->
     <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" >
     <link rel="stylesheet" href="css/style.css">
 
   </head>
   <body>
     <div class="container">
-      <!-- Replace me with your html -->
+<!-- Header -->
+      <header class="Navigation">
+        <div class="box strong">Relaxr</div>
+        <div class="primary-nav">
+          <div class="box">About</a></div>
+          <div class="box">FAQ</div>
+          <div class="box">Team</div>
+          <div class="box">Contact Us</div>
+          <div class="box">Blog</div>
+          <div class="box"></div>
+        </div>
+      </header>
+<!-- Body -->
+      <Section class="flex">
+        <div class="body"></div>  
```

Comment:
instead of adding an extra emtpy element here, use left padding on the other .body element